sycl/include/sycl/accessor.hpp

```diff
@@ -1793,6 +1794,49 @@ class __SYCL_SPECIAL_CLASS accessor :
     return constant_ptr<DataT>(getPointerAdjusted());
   }
 
+  // accessor::has_property for runtime properties is only available in host
+  // code
```

Comment:
Could you please point where it's stated?

Reply:
This is to allow for the `constexpr` variant from the [SYCL_ONEAPI_accessor_properties](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_accessor_properties.asciidoc). That one is a little further down.

Reply:
It is not written in the public domain anywhere. I've added a more detailed comment in fb9beca0b25db3b31f1f975f522fddf205e9330f